recipes/tinycthread/all/conanfile.py

```diff
@@ -0,0 +1,68 @@
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import msvc_runtime_flag, is_msvc
+import functools
+import os
+
+required_conan_version = ">=1.33.0"
+
+class TinycthreadConan(ConanFile):
+    name = "tinycthread"
+    description = "Small, portable implementation of the C11 threads API"
+    license = "zlib"
```

Comment:
```suggestion
    license = "Zlib"
```
https://spdx.org/licenses/Zlib.html